### PR TITLE
Hotfix/quote form style update

### PR DESCRIPTION
**Pull request by Frankruiz90**
- updated styles of the quotation form.
- Update the alignment of the elements of the quotation form.
- The validation function for the quote button was updated.
- unwanted modification of the flatpkr was removed.

### CODE REVIEW

File: css/quotes.css [hunk 1]
```diff
@@ -204,6 +202,11 @@ body {
   border-radius: 50px 0 0 50px;
 }
 
+.button[disabled] {
+  color: gray;
+  background-color: #b4b3b3;
+}
```
Comment (hanna446) [inline css/quotes.css:208]: You changed the color of the button, but the color of the hover is still blue.

**Comment by hanna446** [comment]
The size of the desktop quote button is larger than that of the zeplin design

File: css/quotes.css (hunk 1 above)
Comment (Frankruiz90) [reply css/quotes.css:208]: @hanna446 corrected

**Comment by Frankruiz90** [comment]
> The size of the desktop quote button is larger than that of the zeplin design

@hanna446  size corrected

**Review by jusk02 — APPROVED** [review]

**Review by jhonatanrojasbastidas — APPROVED** [review]

**Pull request merged**